### ISSUE TITLE
Add support for Plymouth

### ISSUE THREAD

**Issue by svartkanin** [archlinux/archinstall#1796]
It is not an essential thing to have to get the system up and running, but considering that setting up Plymouth requires changes in the mkinitcpio.conf it may be much more convenient to offer it through the installer

**Comment by Torxed** [comment]
I agree, I would like to introduce a submenu called "optimizations" that contains non-trivial and community driven options that has zero requirements, but where we can shove popular demands that don't meet the core values of the project.

That way we can still offer the ideas - without being in the way and clutter the experience.

Thought?

**Comment by svartkanin** [comment]
That sounds like a great idea. I know we have the `--advanced` flag - which I think gets overlooked by users quite a bit - would it make sense to put such things under `advanced` in the menu? 

I'm aware that the current advanced flag is for special logic triggers as well not only special menu menu options so it might get more confusing that way

**Comment by Torxed** [comment]
Yea the `--advanced` flag are for those that know and respect the breaking changes it can bring using the features we hide behind the flag.

Proposal:

`--advanced` only purpose is to show a new menu option in the main menu called `experimental`.
A new menu option called `optimizations` or `community features` and we throw non-important features in there - but still vet them and put only those that many want

**Comment by svartkanin** [comment]
That sounds great, i like the idea!

**Comment by gallium-stanza** [comment]
Hi! Are there any recent plans regarding implementing the change discussed above?

**Comment by svartkanin** [comment]
Work hasn't started yet (lots of other things were prioritised), any contributions are more than welcome
